app/Views/errors/html/error_404.php

```diff
@@ -74,10 +74,10 @@
 
 		<p>
 			<?php if (! empty($message) && $message !== '(null)') : ?>
-				<?= nl2br(esc($message)) ?>
+				<?= nl2br(esc($message)); ?>
 			<?php else : ?>
```

Comment:
I think this should be `else:` - do we have a run yet coming for that.

Reply:
I cannot recall if there is.